### DISCUSSION TITLE
Make the border invisible

### DISCUSSION THREAD

**Discussion by sercandemirtas** [google/model-viewer#1936] [Q&A]
Hello all!

How can I make model viewer's border invisible when we are rotating the model. It is blue on Mac and black on Windows. I want to make it invisible. I checked modelviewer.dev and on astronout example there is no border.

Do I need to edit my css codes or is there option on model viewer ? so any idea&suggest would be great. 

Thanks.. 

#### Live Demo
https://unruffled-einstein-e67ebb.netlify.app/urun.html

<img width="1440" alt="Ekran Resmi 2020-11-23 20 42 33" src="https://user-images.githubusercontent.com/68244546/99996266-88679480-2dcc-11eb-9369-be7542bd3462.png">

### Browser Affected

- [ X] Chrome
- [ X] Edge
- [ X] Firefox
- [X ] IE
- [ X] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [X ] MacOS
- [ X] Windows

**Comment by elalish** [comment]
Have you added the focus-visible polyfill? `<script src="../node_modules/focus-visible/dist/focus-visible.js" defer></script>`

**Comment by sercandemirtas** [comment]
<script src="https://unpkg.com/@webcomponents/webcomponentsjs@2.1.3/webcomponents-loader.js"></script>
    <script src="https://unpkg.com/intersection-observer@0.5.1/intersection-observer.js"></script>
    <script src="https://unpkg.com/resize-observer-polyfill@1.5.0/dist/ResizeObserver.js"></script>
    <script src="node_modules/focus-visible/dist/focus-visible.js" defer></script>
    
    <script type="module"
    src="https://unpkg.com/@google/model-viewer@v1.1.0/dist/model-viewer.min.js">
    </script>
  
    <script nomodule
    src="https://unpkg.com/@google/model-viewer@v1.1.0/dist/model-viewer-legacy.js">
    </script>

Hey @elalish I have added your code but nothing changed. (you can check my demo page I shared link) What am I missing ? 


<img width="1440" alt="Ekran Resmi 2020-11-23 22 46 10" src="https://user-images.githubusercontent.com/68244546/100007902-d042e780-2ddd-11eb-9e6a-80408cdd3d8b.png">

**Comment by elalish** [comment]
Sorry, I pasted you a relative URL, try this instead: `<script src="https://unpkg.com/focus-visible@5.0.2/dist/focus-visible.js" defer></script>`

**Comment by sercandemirtas** [comment]
@elalish It worked! Thank you...
